components/molecules/CardHorizontalBarChart/card-horizontal-bar-chart.tsx

```diff
@@ -55,6 +61,11 @@ const CardHorizontalBarChart = ({ languageList, withDescription }: CardHorizonta
               <div
                 key={index}
                 onMouseOver={() => handleChangeDescriptText(languageName)}
+                onFocus={() => handleChangeDescriptText(languageName)}
+                onKeyDown={(e) => handleKeyDown(e, languageName)}
+                tabIndex={0}
+                role="button"
+                aria-label={`${languageName}: ${percentageUsed}%`}
```

Comment:
tabIndex and role props aren't necessary. Use a button instead. Also include button text, i.e. `<span className="sr-only">{`languageName ${percentageUsed}%`}</span>`

```jsx
              <button
                key={index}
                onMouseOver={() => handleChangeDescriptText(languageName)}
                onFocus={() => handleChangeDescriptText(languageName)}
                onKeyDown={(e) => handleKeyDown(e, languageName)}
                aria-label={`${languageName}: ${percentageUsed}%`}
                className="h-2 transition-all duration-500 ease-in-out"
                style={{
                  width: `${percentageUsed < 20 ? (percentageUsed / percentage) * 100 : percentageUsed}%`,
                  backgroundColor: languageToColor[languageName]
                    ? (languageToColor[languageName].color as string)
                    : NOTSUPPORTED,
                }}
              >
                <span className="sr-only">{`languageName ${percentageUsed}%`}</span>
              </button>
```

Reply:
@nickytonline I have implemented the changes you suggested.